### ISSUE TITLE
Orca does not read the line number in the breakpoint view

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#96987]
Issue Type: <b>Bug</b>

1. Launch orca.
2. Launch vscode.
3. Create a file and put some code.
4. Insert breakpoints.
5. Press f1 and chose focus on Breakpoints View.
6. Navigate in the view using arrows.

Orca only announces the name of the file without announcing the line number, although there appears to be a line number in the view.
cc/ @isidorn 
VS Code version: Code - Insiders 1.45.0-insider (d4b8ec9da3bf1061a948fbb1b0d9d3db750cc79f, 2020-05-05T07:09:48.406Z)
OS version: Linux x64 5.6.10-arch1-1

<details><summary>Extensions (23)</summary>

Extension|Author (truncated)|Version
---|---|---
prettier-vscode|esb|4.5.0
rest-client|hum|0.23.2
code-groovy|mar|0.1.2
xml-format|mik|1.0.2
vscode-docker|ms-|1.1.0
csharp|ms-|1.21.17
python|ms-|2020.4.76186
remote-containers|ms-|0.115.0
remote-ssh|ms-|0.51.0
remote-ssh-edit|ms-|0.51.0
remote-wsl|ms-|0.44.2
vscode-remote-extensionpack|ms-|0.20.0
sqltools|mtx|0.21.7
gradle-language|nac|0.2.3
java|red|0.61.0
vscode-xml|red|0.11.0
vscode-checkstyle|she|1.3.2
vscodeintellicode|Vis|1.2.6
vscode-java-debug|vsc|0.25.1
vscode-java-dependency|vsc|0.9.0
vscode-java-pack|vsc|0.9.0
vscode-java-test|vsc|0.22.3
vscode-maven|vsc|0.21.2

</details>

**Comment by isidorn** [comment]
@jvesouza great catch. It is fixed now, note that I only append the acutal number. I do not see "line number 7", just "7". If you would like a more concise form do please let me know.
Also great that you are using the debuger, let me know how it is accessibility wise!
This week we are releasing our stable build so the insdiers will only have the update with this fix on Friday I think
Thanks a lot

**Comment by jvesouza** [comment]
Thanks for the fix.

I started using the debugger recently in java applications and until now it seemed very accessible.

The problems I've encountered so far seem to have already been reported in issuee # 90842.

**Comment by isidorn** [comment]
Glad to hear that, thanks!

**Comment by jvesouza** [comment]
This is fixed, great job.
Thanks.

**Comment by isidorn** [comment]
Thanks for letting us know, adding verified label.
